src/Presentation/Nop.Web/App_Data/Localization/defaultResources.nopres.xml

```diff
@@ -17256,6 +17256,9 @@
   <LocaleResource Name="Products.Qty">
     <Value>Qty</Value>
   </LocaleResource>
+  <LocaleResource Name="Products.Qty.AriaLabel">
+    <Value>Enter a quantity</Value>
+  </LocaleResource>
```

Comment:
The value of the new locale must also be added to the update script (at the end of the list). This is done in order to be able to apply your changes without reinstalling the store.
_/upgradescripts/4.20-4.30 (under development)/upgrade.sql_